lib/caasp.pm

```diff
@@ -33,6 +33,11 @@ sub process_reboot {
     script_run("reboot", 0) if $trigger;
 
     reset_consoles;
+    if (check_var('VIRSH_VMM_FAMILY', 'xen')) {
+        sleep 7;
+        select_console 'sut';
+    }
```

Comment:
I'd rather not use `wait_boot` for non-SLES/openSUSE distros at all, as it's really not needed and only adds complexity. And sure we need a 'reconnect' solution in os-autoinst; will file ticket about that.